node-notes-cli/update.js

```diff
@@ -0,0 +1,16 @@
+
+const fs = require('fs');
+const json = require('./data.json');
+
+const update = (enter, enter2) => {
+  json.notes[enter] = enter2;
+  const final = JSON.stringify(json, null, 2);
+  fs.writeFile('data.json', final, 'utf8', err => {
+    if (err) {
+      throw err;
+    }
+
+  });
+};
+
+module.exports = update;
```

Comment:
"User can update a note" should only be able to modify existing notes, not create new ones.

---

![image](https://user-images.githubusercontent.com/16143771/150584747-1a4e5218-36dd-4927-8aff-d448e8f44ea8.png)